static/app/views/eventsV2/table/index.tsx

```diff
@@ -162,6 +164,16 @@ class Table extends PureComponent<TableProps, TableState> {
           pageLinks: resp ? resp.getResponseHeader('Link') : prevState.pageLinks,
           tableData,
         }));
+
+        const tips: string[] = [];
+        const {query, columns} = tableData?.meta?.tips ?? {};
+        if (query) {
+          tips.push(query);
+        }
+        if (columns) {
+          tips.push(columns);
+        }
+        setTips?.(tips);
```

Comment:
Hmm do we not need the distinction between tip types? if so we could just send a single list in the API instead of differentiating between column and query

Reply:
For now I'm just displaying query and column tips in the same way as an alert.
I haven't come up with an idea on how to separate query and column tips ui wise yet but that possibility is definitely still open. Do you think this is something we should tackle in this pr? If we're not eventually going to separate query and column tips, then yeah I also agree we could just send all tips as a single list from the api

Reply:
Doesn't need to be tackled in this PR :+1: lets get this out there and see if more people use IN queries over a chain of ORs